src/ui/views/task-view.tsx

```diff
@@ -107,8 +107,38 @@ function TaskItem({ item }: { item: STask }) {
 
 /** JSX component which renders a plain list item recursively. */
 function ListItem({ item }: { item: SListEntry }) {
+    let context = useContext(DataviewContext);
+    
+    // Navigate to the given task on click.
+    const onClicked = (evt: preact.JSX.TargetedMouseEvent<HTMLElement>) => {
+        // Skip this event if a link was pressed.
+        if (evt.target != null && evt.target != undefined && (evt.target as HTMLElement).tagName == "A") {
```

Comment:
```suggestion
        if (wasLinkPressed(evt)) {
            return 
        }
```
would be a cleaner way to express that idea by neatly refactoring the check into its own function. Removes the need for the comment. :)

Reply:
> Could you also reset the changes to the test vault?
Sure give me 5 mins.

> would be a cleaner way to express that idea by neatly refactoring the check into its own function. Removes the need for the comment. :)
Hehe I ugly copy/pasted that code from `TaskItem` component. Your approach seems indeed cleaner, thanks :)